src/AdminDashboard/SearchEngine/computerOptions.json

```diff
@@ -0,0 +1,1989 @@
+{
```

Comment:
Is the existence of this file really necessary? Is there no way to bring this data asynchronously from the api-rest?

Reply:
The glpi javascript client does not have the method to obtain the fields of each itemType, I already sayed it to @ajsb85  and he told me to create an issue.

Reply:
https://github.com/glpi-project/glpi/blob/9.2/bugfixes/apirest.md#list-searchoptions